js/forum/src/mutateUserBio.js

```diff
@@ -1,28 +1,24 @@
 import {override} from "flarum/extend";
 import UserBio from "flarum/components/UserBio";
-import icon from "flarum/helpers/icon";
-import Mutate from "flagrow/masquerade/utils/Mutate";
+import TypeFactory from 'flagrow/masquerade/types/TypeFactory';
 
 export default function () {
     override(UserBio.prototype, 'view', function (view) {
         // Load the old user bio.
         let original = app.forum.attribute('masquerade.disable-user-bio') ? null : view();
         let answers = app.forum.attribute('canViewMasquerade') ? this.props.user.bioFields() || [] : [];
 
-        return m('div', {className: 'Masquerade-Bio'}, [
+        return m('.Masquerade-Bio', [
```

Comment:
What happened with the element here?

Reply:
I used the shortcut notation to create a `div` with class `Masquerade-Bio`, I think this is a lot cleaner visually.

But that's a format preference we could discuss :yum:

Reply:
I wasn't aware about that. Good to know. Please leave as is.